packages/manager/apps/dedicated/client/app/networkSecurity/network-security.html

```diff
@@ -0,0 +1,34 @@
+<div data-ui-view="networkSecurityView">
+    <div data-ovh-alert="network_security_error"></div>
+    <oui-header
+        data-heading="{{:: 'network_security_dashboard_title' | translate }}"
+    >
+        <oui-guide-menu
+            text="{{:: 'network_security_dashboard_guides' | translate }}"
+        >
+            <oui-guide-menu-item href="#" external>
+                External link
```

Comment:
**question:** Dont we need translation for this?

Reply:
For the moment I don't have the link to the guide. It's to remember to replace it.